posthog/models/feedback/survey.py

```diff
@@ -277,7 +277,10 @@ def update_survey_iterations(sender, instance, *args, **kwargs):
     instance.iteration_start_dates = list(
         rrule(
             DAILY,
-            count=iteration_count,
+            # we have seen users accidentally set a huge range here
+            # and cause performance issues, so we are extra careful with this value
+            # NB this is enforced in the UI too
+            count=max(iteration_count, 500),
```

Comment:
Dont you want the min? The smallest number between 500 and the provided value?

Reply:
yep ❤️  luckily the tests caught this brain fart 🙈

Reply:
good catch =D